Fix:
  NameError:
    uninitialized constant Tempfile
  # ./spec/rmagick/image_list/write_spec.rb:39:in `block (2 levels) in <top (required)>'

--- a/spec/rmagick/image_list/write_spec.rb
+++ b/spec/rmagick/image_list/write_spec.rb
@@ -1,3 +1,5 @@
+require 'tempfile'
+
 RSpec.describe Magick::ImageList, "#write" do
   it "works" do
     image_list = described_class.new
